catalog-info.yaml

```diff
@@ -32,3 +35,35 @@ spec:
         everyone:
           access_level: READ_ONLY
         search-productivity-team: {}
+
+---
+# ------------------------------------------------------------------------------
+# docker image build and publish - manual release
+
+apiVersion: "backstage.io/v1alpha1"
+kind: "Resource"
+metadata:
+  name: "crawler-docker-build-publish"
+  description: "Docker image build and publish for Elastic crawler"
+  links:
+    - title: "Crawler Docker Build and Publish"
+      url: "https://buildkite.com/elastic/crawler-docker-build-publish"
+spec:
+  type: "buildkite-pipeline"
+  owner: "group:ingestion-team"
+  system: "buildkite"
+  implementation:
+    apiVersion: "buildkite.elastic.dev/v1"
+    kind: "Pipeline"
+    metadata:
+      name: "crawler-docker-build-publish"
+    spec:
+      repository: "elastic/crawler"
+      pipeline_file: ".buildkite/release-pipeline.yml"
```

Comment:
This file might need to actually exist? But we'll find out. 🤞

Reply:
pretty sure this won't work without that file, but could be wrong

maybe easiest to merge https://github.com/elastic/crawler/pull/103 even if it's not complete/done and then iterate on it from there

Reply:
I think I'll just focus on #103, as I don't want to spend more effort on this PR if it's more than just catalog-info.yml.
I'm sure something will be automatically spun up for us (and if not, we can troubleshoot from there)

Reply:
I think we could have merged this as it was, and/or could have added a `.buildkite/release-pipeline.yml` with a single step of `echo 'hello world'`.

The bummer about not having this merged is that you can't test the pipeline in 103 without merging it first. Which is generally not how we like to do things. I don't think there's much risk in this, so Nav if you're fine merging 103 _whenever_, then trying it out and fixing issues in a follow-up PR, that's fine. If you'd rather all the real changes be in one PR, I think you might want to come back to this approach. Either is fine.

Reply:
Yeah that makes sense. No idea what I was thinking yesterday. I think I was just trying to reduce PRs.